docs/getting_started/account_security.md

```diff
@@ -60,11 +60,9 @@ To disable two-factor authentication on your SEKOIA account:
 4. Enter your actual password and select `disable`
 
 ## Security tokens
-In SEKOIA.IO, we support Web Authentication and we value high security standards. That’s why our users can use multiple tools to sign in on the platform in the fastest and most secure way possible.
+Web Authentication is supported in SEKOIA.IO as we value high security standards. Users can use multiple tools to access their account in the fastest and most secure way possible. These security tokens can be used as a second factor of authentication instead of a verification code.
```

Comment:
```suggestion
WebAuthn standard (Web Authentication) is supported in SEKOIA.IO as we value high security standards. Users can use multiple tools to access their account in the fastest and most secure way possible. These security tokens can be used as a second factor of authentication instead of a verification code.
```